examples/contrastive-image-text/clip_media_pipe.py

```diff
@@ -128,7 +128,9 @@ def __next__(self):
     read_image_text_from_dataset,
     dtype.NDT,
 )
-op_class = fn.operator_add("ClipDataReader")
+#TODO : NEED to add version check and have backward compatible
+op_class = fn.operator_add("ClipDataReader", False)
+#op_class = fn.operator_add("ClipDataReader")
```

Comment:
@regisss  Habana FW api changes causing the error for bridgetower.  Need to add version check to have both 1.15.0 and 1.14.0 to work.

Reply:
Yep, here is an example of version check: https://github.com/huggingface/optimum-habana/blob/20832c92ce3e9cfe94aeb1797a5533b4be962a6c/examples/text-generation/utils.py#L177

Reply:
I'm going to push a commit for it

Reply:
@jiminha WDYT? https://github.com/huggingface/optimum-habana/pull/776/commits/8bbe1173581e27f18f1d052472e6a1f4d26bb888

Reply:
Validated both on 15.0 and 14.0.  No error seen.